core/lib/eth_signer/src/raw_ethereum_tx.rs

```diff
@@ -47,6 +48,10 @@ pub struct TransactionParameters {
     pub max_fee_per_gas: U256,
     /// miner bribe
     pub max_priority_fee_per_gas: U256,
+    /// Max fee per blob gas
```

Comment:
These comments don't help much ...

Maybe instead say that for 4844 blobs, are using separate fee market.
And mention what happens if blobs are present and this is not set and vice versa -- if I have blobs, does this have to be set too?

Reply:
Expanded a comment a bit